README.md

```diff
@@ -5,8 +5,9 @@ This repo holds generated metadata about the Flutter framework.
 The metadata is useful for Flutter related tooling and is currently used by
 Flutter IDE tools such as the Flutter IntelliJ and VS Code plugins.
 
-If you're using this metadata, please let us know! You can open an issue on the
-repo or add your tool to this readme via a pull request.
+If you're using this metadata, please let us know! You can open an [issue] on the
+repo (please use the `d: tools_metadata` label) or add your tool to this readme
```

Comment:
I don't think most users can assign labels directly themselves (except maybe via a template?)

Reply:
Yeah, I don't think so either, but at least mentioning the label here might help them w/ triage or the issue description.